docker-compose.yaml

```diff
@@ -5,30 +5,46 @@ services:
     environment:
-      - SERVERFULL_RUNTIME_HTTPSERVER_ADDRESS=:8081
-      - SERVERFULL_RUNTIME_CONNSTATE_REPORTINTERVAL=5s
-      - SERVERFULL_RUNTIME_CONNSTATE_HIJACKEDCOUNTER=http.server.connstate.hijacked
-      - SERVERFULL_RUNTIME_CONNSTATE_CLOSEDCOUNTER=http.server.connstate.closed
-      - SERVERFULL_RUNTIME_CONNSTATE_IDLEGAUGE=http.server.connstate.idle.gauge
-      - SERVERFULL_RUNTIME_CONNSTATE_IDLECOUNTER=http.server.connstate.idle
-      - SERVERFULL_RUNTIME_CONNSTATE_ACTIVEGAUGE=http.server.connstate.active.gauge
-      - SERVERFULL_RUNTIME_CONNSTATE_ACTIVECOUNTER=http.server.connstate.active
-      - SERVERFULL_RUNTIME_CONNSTATE_NEWGAUGE=http.server.connstate.new.gauge
-      - SERVERFULL_RUNTIME_CONNSTATE_NEWCOUNTER=http.server.connstate.new
-      - SERVERFULL_RUNTIME_LOGGER_OUTPUT=STDOUT
-      - SERVERFULL_RUNTIME_LOGGER_LEVEL=INFO
-      - SERVERFULL_RUNTIME_STATS_OUTPUT=DATADOG
-      - SERVERFULL_RUNTIME_STATS_DATADOG_PACKETSIZE=32768
-      - SERVERFULL_RUNTIME_STATS_DATADOG_TAGS=
-      - SERVERFULL_RUNTIME_STATS_DATADOG_FLUSHINTERVAL=10s
-      - SERVERFULL_RUNTIME_STATS_DATADOG_ADDRESS=statsd:8125
-      - SERVERFULL_RUNTIME_SIGNALS_INSTALLED=OS
-      - SERVERFULL_RUNTIME_SIGNALS_OS_SIGNALS=15 2
-      - VULNERABILITYFILTER_CVSSV2MINIMUMSCORE=7.0
-      - VULNERABILITYFILTER_VULNIDREGEXMATCH=tls-server-cert.*expire.*|ssl-.*sign.*
-  gateway:
+      SERVERFULL_RUNTIME_HTTPSERVER_ADDRESS: :8081
+      SERVERFULL_RUNTIME_CONNSTATE_REPORTINTERVAL: 5s
+      SERVERFULL_RUNTIME_CONNSTATE_HIJACKEDCOUNTER: http.server.connstate.hijacked
+      SERVERFULL_RUNTIME_CONNSTATE_CLOSEDCOUNTER: http.server.connstate.closed
+      SERVERFULL_RUNTIME_CONNSTATE_IDLEGAUGE: http.server.connstate.idle.gauge
+      SERVERFULL_RUNTIME_CONNSTATE_IDLECOUNTER: http.server.connstate.idle
+      SERVERFULL_RUNTIME_CONNSTATE_ACTIVEGAUGE: http.server.connstate.active.gauge
+      SERVERFULL_RUNTIME_CONNSTATE_ACTIVECOUNTER: http.server.connstate.active
+      SERVERFULL_RUNTIME_CONNSTATE_NEWGAUGE: http.server.connstate.new.gauge
+      SERVERFULL_RUNTIME_CONNSTATE_NEWCOUNTER: http.server.connstate.new
+      SERVERFULL_RUNTIME_LOGGER_OUTPUT: STDOUT
+      SERVERFULL_RUNTIME_LOGGER_LEVEL: INFO
+      SERVERFULL_RUNTIME_STATS_OUTPUT: DATADOG
+      SERVERFULL_RUNTIME_STATS_DATADOG_PACKETSIZE: 32768
+      SERVERFULL_RUNTIME_STATS_DATADOG_TAGS:
+      SERVERFULL_RUNTIME_STATS_DATADOG_FLUSHINTERVAL: 10s
+      SERVERFULL_RUNTIME_STATS_DATADOG_ADDRESS: statsd:8125
+      SERVERFULL_RUNTIME_SIGNALS_INSTALLED: OS
+      SERVERFULL_RUNTIME_SIGNALS_OS_SIGNALS: 15 2
+      NEXPOSEVULNFILTER_LAMBDAMODE: "false"
+      NEXPOSEVULNFILTER_PRODUCER_TYPE: BENTHOS
+      NEXPOSEVULNFILTER_PRODUCER_BENTHOS_YAML: ""
+      NEXPOSEVULNFILTER_PRODUCER_POST_ENDPOINT: ""
+      NEXPOSEVULNFILTER_PRODUCER_POST_HTTPCLIENT_TYPE: "DEFAULT"
+      NEXPOSEVULNFILTER_PRODUCER_POST_HTTPCLIENT_SMART_OPENAPI: ""
+      NEXPOSEVULNFILTER_CVSSV2MINIMUMSCORE: 7.0
```

Comment:
Task: with the naming convention you've defined, these should be `NEXPOSEVULNFILTER_VULNERABILITYFILTER_[CVSSV2MINIMUMSCORE|VULNIDREGEXMATCH]`

Reply:
Is that behavior documented anywhere? I didn't see it while quickly scanning through the docs and source for the settings library.

Reply:
In any case, thanks, I'll fix this. 👍

Reply:
Thanks! Missed that.